### PR TITLE
FFWEB-1606: Added information about `channel` to `API` and `Documentition` section

### PR DESCRIPTION
**Pull request by a-laurowski**

### CODE REVIEW

File: markdown/3.x/en/api/ff-checkout-tracking.api.md [hunk 1]
```diff
@@ -20,3 +20,4 @@ ___
 | **record-id**&nbsp;(String) (default: empty) | The item's ID used for tracking in FACT-Finder. |
 | **count**&nbsp;(String) (default: "1") | Number of pieces of this item that were bought. |
 | **price**&nbsp;(String) | Price of the item bought. If omitted, it will be inferred from the clicked item and the configured [fieldRoles](/documentation/3.x/field-roles). |
+| **channel**&nbsp;(String) | Channel used to send a tracking request. If omitted, default value configured in `ff-communication` will be used.
```
Comment (omikron-fp) [inline markdown/3.x/en/api/ff-checkout-tracking.api.md]:
```suggestion
| **channel**&nbsp;(String) | Channel used to send a tracking request. If omitted, the default value configured in `ff-communication` will be used.
```

File: markdown/3.x/en/ff-checkout-tracking.md [hunk 2]
```diff
@@ -10,9 +10,12 @@ The `count` attribute specifies how many items of the product were bought. It de
 
 The `price` attribute specifies the price of the bought item. If this attribute is not present, it will be inferred from the clicked item and the configured [fieldRoles](/documentation/3.x/field-roles).
 
+The `channel` attribute specifies the channel, tracking request should be send to. If this attribute is not present, default value configured in `ff-communication` will be used.
```
Comment (omikron-fp) [inline markdown/3.x/en/ff-checkout-tracking.md]:
```suggestion
The `channel` attribute specifies the channel to which a tracking request should be sent. If this attribute is not present, the default value configured in `ff-communication` will be used.
```

File: markdown/3.x/en/ff-checkout-tracking.md [hunk 3]
```diff
@@ -10,9 +10,12 @@ The `count` attribute specifies how many items of the product were bought. It de
...
+
+
 ```HTML
 <ff-checkout-tracking>
-    <ff-checkout-tracking-item record-id="10321926" count="5"></ff-checkout-tracking-item>
+    <ff-checkout-tracking-item record-id="10321926" count="5" channel="NEW_CHANNEL"></ff-checkout-tracking-item>
```
Comment (omikron-fp) [inline markdown/3.x/en/ff-checkout-tracking.md]: In the _Tracking Guide_ it is `channel="[NEW_CHANNEL]"`. For consistency I would suggest to remove `[ ]` in the _Tracking Guide_.

Comment (a-laurowski) [reply markdown/3.x/en/ff-checkout-tracking.md]: changed

**Pull request merged**